apps/web/src/server/app/auth/auth.controller.js

```diff
@@ -1 +1,29 @@
-/* eslint-disable-line unicorn/no-empty-file */
+// @ts-ignore
+import { ssoConfig } from '../../config/sso.js';
+import { cca } from '../../lib/sso.js';
+
+/**
+ * Path to redirect to after successful login.
+ *
+ * @type {import('@pins/express').QueryHandler<{}>}
```

Comment:
I think it would be useful to define the query params, as I'm not sure it's clear what "code" is...
```
/**
 * @typedef {object} AuthRedirectQuery
 * @property {string} code - <What this is>
 */
```
and then you can plug it in via:
```
/** @type {import('express').RequestHandler<?, ?, ?, AuthRedirectQuery>} */
```
(You could use the @pins/express handler but it's primarily for handling routes rendering html)